src/builders/dev-server/index.ts

```diff
@@ -1,12 +1,14 @@
 import { DevServerBuilder } from '@angular-devkit/build-angular';
 import { BuilderContext } from '@angular-devkit/architect';
-import { virtualFs, Path, tags} from '@angular-devkit/core';
+import { virtualFs, Path, tags } from '@angular-devkit/core';
 import { SingleSpaBuilderSchema } from '../browser/schema';
 import { Configuration } from 'webpack';
 import * as fs from 'fs';
+import * as path from 'path';
 import { SingleSpaBrowserBuilder } from '../browser';
+import * as webpackMerge from 'webpack-merge';
 
-
+// @ts-ignore
```

Comment:
The `_buildServerConfig` function is the only way to override angular's default dev server configuration. But typescript freaks out because it's a private method, so I disabled that check.